internal/resources/data_source_ssl_certs.go

```diff
@@ -0,0 +1,52 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func LBVirtualServerSslCertData() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: f(generalNamedesc, "LoadBalancer", "LoadBalancer"),
```

Comment:
is LoadBalancer enough ? Can you check with other data source such as NAT ? please make this change whereever needed. This would be shown in doc later

Reply:
updated